### PR TITLE
navbar and husky

### PR DESCRIPTION
**Pull request by meenachan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +20,13 @@
     "lighthouse": "node ./jobs/lighthouse.js",
     "dev": "next dev",
     "build": "next build",
-    "start": "next start -p $PORT"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
+    "start": "next start -p $PORT",
+    "lint": "npx eslint ./ --fix",
```
Comment (meShakti) [inline package.json]: remove --fix in lint command

File: package.json [hunk 2]
```diff
@@ -18,11 +20,13 @@
...
+    "test:lint:fix": "npx eslint ./ --fix",
+    "test:lint:css": "stylelint  \"**/*.css\" --fix",
```
Comment (meShakti) [inline package.json]: rename test:lint:fix to lint:fix . do the same with css and rename test command to lint:all

File: components/layout/index.js [hunk 3]
```diff
@@ -1,19 +1,64 @@
...
+const Layout = ({ children }) => {
+	
+	const [isExpanded, toggleExpansion] = useState(false);
+  
+  
+	return (
+
+		<div className={layout.layout}>
+
+			<nav className="flex items-center justify-between flex-wrap bg-black p-6">
```
Comment (meShakti) [inline components/layout/index.js]: Make a header component and move this logic there

File: components/layout/index.module.css [hunk 4]
```diff
@@ -16,4 +15,9 @@
     text-transform: uppercase;
     text-align:center;
     padding: 3rem;
+}
+.overlaysym{
```
Comment (meShakti) [inline components/layout/index.module.css]: if this is related to header move this to header.module.css and make this classname meaningful

File: jobs/lighthouse.js [hunk 5]
```diff
@@ -1,19 +1,19 @@
-const lighthouse = require('lighthouse');
-const chromeLauncher = require('chrome-launcher');
-const fs = require('fs');
-const {exec} = require('child_process');
+const lighthouse = require("lighthouse");
```
Comment (meShakti) [inline jobs/lighthouse.js:1]: Why these changes are made ? is this required

File: package.json [hunk 6]
```diff
@@ -34,5 +38,46 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "eslintIgnore": [
+    "*.json",
+    "jobs/*.js"
+  ],
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^3.7.1",
```
Comment (meShakti) [inline package.json]: validate once if all the depenencies are required . we are not using typescript, why ts related dependencies are there

File: pages/index.js [hunk 7]
```diff
@@ -1,102 +1,103 @@
-import Layout from '../components/layout';
-import ProductList from '../components/product/list';
+import React from "react";
+import Layout from "../components/layout";
+import ProductList from "../components/product/list";
 
 const productList = [
```
Comment (meShakti) [inline pages/index.js]: Move this varaialbe in a data.js file in data folder in source  aand import  it from there

File: components/layout/index.js [hunk 8]
```diff
@@ -1,19 +1,64 @@
...
+					<span className="font-semibold text-xl tracking-tight">New Abhishek</span>
+				</div>
+
+				<div className="block lg:hidden">
+					<button onClick={() => toggleExpansion(!isExpanded)} className="flex items-center px-3 py-2 border rounded text-teal-200 border-teal-400 hover:text-white hover:border-white">
+						<svg className="fill-current h-3 w-3" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><title>Menu</title><path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z" /></svg>
+					</button>
+				</div>
+				<div className={`${isExpanded ? "block" : "hidden"} w-full block flex-grow lg:flex lg:items-center lg:w-auto`}>
+					<div className="lg:flex lg:items-stretch lg:justify-end ml-auto">
```
Comment (meShakti) [inline components/layout/index.js]: make sure the clases in flex are available in tailwind because this is what we are using in the project

File: components/layout/index.js (hunk 3 above)
Comment (meenachan) [reply components/layout/index.js]: Done

File: package.json (hunk 1 above)
Comment (meenachan) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (meenachan) [reply package.json]: done

File: components/layout/index.js [hunk 9]
```diff
@@ -1,19 +1,63 @@
...
+
+const Layout = ({ children }) => {
+	
+	const [isExpanded, toggleExpansion] = useState(false);
+  
+  
+	return (
+
+		<div className={layout.layout}>
+           <Header></Header>
```
Comment (meShakti) [inline components/layout/index.js]: Use self Closing tag

File: components/layout/index.js [hunk 10]
```diff
@@ -1,19 +1,63 @@
...
+			{/* <nav className="flex items-center justify-between flex-wrap bg-black p-6">
```
Comment (meShakti) [inline components/layout/index.js]: Remove Comment

File: components/Header/Header.js [hunk 11]
```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
```
Comment (meShakti) [inline components/Header/Header.js:1]: Make header.module.css and use it instead of index.module.css

File: package.json [hunk 12]
```diff
@@ -18,11 +20,14 @@
     "lighthouse": "node ./jobs/lighthouse.js",
     "dev": "next dev",
     "build": "next build",
-    "start": "next start -p $PORT"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
+    "start": "next start -p $PORT",
+    "lint": "npx eslint ./ ",
+    "test:lint:fix": "npx eslint ./ --fix",
```
Comment (meShakti) [inline package.json]: remove unused commands

File: components/Header/Header.module.css [hunk 13]
```diff
@@ -0,0 +1,5 @@
+.cartitembubbles{
```
Comment (meShakti) [inline components/Header/Header.module.css]: use .cart_title_bubble

File: package.json (hunk 6 above)
Comment (meShakti) [reply package.json]: @meenachan  please take care of this

**Review by meShakti — APPROVED** [review]
